backend/ventserver/io/fileobject.py

```diff
@@ -0,0 +1,15 @@
+""" Generic support for File I/O connections.
+TODO: Change name of the module"""
+
+import os
+from typing import Text
+
+import attr
+
+@attr.s(auto_attribs=True)
+class FileProps:
+    """ Defines protobuf file details."""
+
+    filename: str = "state.pb"
+    filedir: str = os.path.join(os.getcwd(), "ventserver", "statestore")
```

Comment:
I am thinking it might be helpful to make the file handler to take a path to a root directory (which defaults to `os.path.join(os.getcwd(), "ventserver", "statestore")` but can be set to something else), and then `FileProps.filedir` specifies the child directory path within the root directory (and maybe it defaults to `""`). This way, we can change where the root directory is more easily (since in production mode it will need to be on a USB drive).